### PR TITLE
Twitter, Instagram  Faith in Humanity stats

### PR DESCRIPTION
**Pull request by impshum**

### CODE REVIEW

**Comment by gingerbeardman** [comment]
Please solve the outstanding Travis CI issue(s) so this can be merged! Many thanks.

You can close and reopen the PR to get Travis to try again, see #1214

File: Web/Instagram/igs.10m.py [hunk 1]
```diff
@@ -1,20 +1,18 @@
 #!/usr/bin/python
```
Comment (tresni) [inline Web/Instagram/igs.10m.py]: This file should not be included in the changes.

File: Web/fih.12h.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
```
Comment (tresni) [inline Web/fih.12h.py]: Please only submit either this file or the faith_in_humanity_score.6h.py file.

**Comment by tresni** [comment]
This PR appears to only add renaming the file with a time interval (15m).  Will do that and commit

**Pull request closed without merging**